mkdocs.yml

```diff
@@ -29,6 +29,7 @@ theme:
 
 nav:
   - Home: index.md
+  - 'Antares ecosystem' : 'https://antares-doc.readthedocs.io'
```

Comment:
There is already this link after the develop part.
I guess it's better to put it afterward because the user coming here wants foremost to see the antares web doc